csharp/src/sdk/Geometry/Polygon.cs

```diff
@@ -161,81 +111,36 @@ public Vector3 PointAt(double u)
         }
 
         /// <summary>
-        /// Tessellate the polygon.
-        /// </summary>
-        /// <returns></returns>
-        public IEnumerable<Vector3> Tessellate()
-        {
-            return this._vertices;
-        }
-
-        /// <summary>
-        /// Reverse the direction of a polygon.
-        /// </summary>
-        /// <returns>Returns a new polgon with opposite winding.</returns>
-        public Polygon Reversed()
-        {
-            var verts = new List<Vector3>(_vertices);
-            verts.Reverse();
-            return new Polygon(verts);
-        }
-    }
-
-    /// <summary>
-    /// Polygon extension methods.
-    /// </summary>
-    internal static class PolygonExtensions
-    {
-        internal enum PolygonOps
-        {
-            Difference,
-            Intersection,
-            Union,
-            XOR
-        }
-
-        /// <summary>
-        /// Construct a clipper path from a Polygon.
+        /// Returns true if every vertex of the supplied Polygon falls within the perimeter of this Polygon.
         /// </summary>
-        /// <param name="p"></param>
+        /// <param name="polygon">The Polygon to compare.</param>
         /// <returns></returns>
-        internal static List<IntPoint> ToClipperPath(this Polygon p)
+        public bool Contains(Polygon polygon)
         {
-            var scale = 1024.0;
-            var path = new List<IntPoint>();
-            foreach(var v in p.Vertices)
+            var thisPath = PolygonExtensions.ToClipperPath(this);
```

Comment:
Oh. Duh. Okay.

Reply:
Seems like if I want to send the supplied Polygon ToClipperPath I need to have the PolygonExtensions. prefix. Otherwise VS tells me there's no overload that accepts one argument.

Reply:
The this. syntax works fine for the Polygon itself.